camera-core/api/camera-core.api

```diff
@@ -82,38 +37,38 @@ public final class com/stripe/android/camera/framework/util/MemoizeKt {
 	public static final fun cachedFirstResultSuspend (Lkotlin/jvm/functions/Function2;)Lkotlin/jvm/functions/Function2;
 	public static final fun cachedFirstResultSuspend (Lkotlin/jvm/functions/Function3;)Lkotlin/jvm/functions/Function3;
 	public static final fun cachedFirstResultSuspend (Lkotlin/jvm/functions/Function4;)Lkotlin/jvm/functions/Function4;
-	public static final fun memoize (Lcom/stripe/android/camera/framework/time/Duration;Lkotlin/jvm/functions/Function0;)Lkotlin/jvm/functions/Function0;
-	public static final fun memoize (Lcom/stripe/android/camera/framework/time/Duration;Lkotlin/jvm/functions/Function1;)Lkotlin/jvm/functions/Function1;
-	public static final fun memoize (Lcom/stripe/android/camera/framework/time/Duration;Lkotlin/jvm/functions/Function2;)Lkotlin/jvm/functions/Function2;
-	public static final fun memoize (Lcom/stripe/android/camera/framework/time/Duration;Lkotlin/jvm/functions/Function3;)Lkotlin/jvm/functions/Function3;
 	public static final fun memoize (Lkotlin/jvm/functions/Function0;)Lkotlin/jvm/functions/Function0;
 	public static final fun memoize (Lkotlin/jvm/functions/Function1;)Lkotlin/jvm/functions/Function1;
 	public static final fun memoize (Lkotlin/jvm/functions/Function2;)Lkotlin/jvm/functions/Function2;
 	public static final fun memoize (Lkotlin/jvm/functions/Function3;)Lkotlin/jvm/functions/Function3;
-	public static final fun memoizeSuspend (Lcom/stripe/android/camera/framework/time/Duration;Lkotlin/jvm/functions/Function1;)Lkotlin/jvm/functions/Function1;
-	public static final fun memoizeSuspend (Lcom/stripe/android/camera/framework/time/Duration;Lkotlin/jvm/functions/Function2;)Lkotlin/jvm/functions/Function2;
-	public static final fun memoizeSuspend (Lcom/stripe/android/camera/framework/time/Duration;Lkotlin/jvm/functions/Function3;)Lkotlin/jvm/functions/Function3;
-	public static final fun memoizeSuspend (Lcom/stripe/android/camera/framework/time/Duration;Lkotlin/jvm/functions/Function4;)Lkotlin/jvm/functions/Function4;
+	public static final fun memoize-VtjQ1oo (JLkotlin/jvm/functions/Function0;)Lkotlin/jvm/functions/Function0;
```

Comment:
There are still a bunch here. Can you make them disappear?

Reply:
RIP i can't believe I missed these.